src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/Touch/TouchBehavior.methods.shared.cs

```diff
@@ -63,7 +63,7 @@ internal async Task ForceUpdateState(CancellationToken token, bool animated = tr
 		}
 		catch (TaskCanceledException ex)
 		{
-			Trace.WriteLine(ex);
+			Trace.TraceError(ex.StackTrace);
 		}
```

Comment:
For this we might just want to report the Message. For a cancellation it could result in a lot of text being logged which just adds noise